plugins/source/terraform/client/backend.go

```diff
@@ -21,9 +21,9 @@ type BackendType string
 
 // BackendConfigBlock - abstract backend config
 type BackendConfigBlock struct {
-	BackendName string                 `yaml:"name"`
-	BackendType string                 `yaml:"backend"`
-	ConfigAttrs map[string]interface{} `yaml:",inline"`
+	BackendName string                 `json:"name"`
+	BackendType string                 `json:"backend"`
+	ConfigAttrs map[string]interface{} `json:",inline"`
```

Comment:
This will need testing.

Reply:
Fixed, the new config format has the backend-specific config under backend-specific keys.